src/shared/adjusters/messages/en/increaseSize.json

```diff
@@ -6,5 +6,7 @@
     "additionalMagnifierAdjusters": "additional magnifier preferences",
     "additionalVisualAdjusters": "additional visual preferences",
     "moreMagnifierPreferences": "more magnifier preferences",
-    "lessMagnifierPreferences": "less magnifier preferences"
-}
\ No newline at end of file
+    "lessMagnifierPreferences": "less magnifier preferences",
```

Comment:
I guess the "more" and "less" came from the UI labels/buttons in the PMT -  which reveal more and less of the tool.